### PR TITLE
feature/demo-changes: Added temp json roles file, added 2 demo regist…

### PR DESCRIPTION
**Pull request by AltusBarry**
…er templates, updated claims to return json roles

### CODE REVIEW

File: authentication_service/demo/roles.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
+    "client_id_1": {
+        "": ["Admin", "Editor", "Trainee", "Manager"]
```
Comment (cobusc) [inline authentication_service/demo/roles.json]: Blank key?

Comment (AltusBarry) [reply authentication_service/demo/roles.json]: Will need to be added locally as users get created during the demo.

Comment (cobusc) [reply authentication_service/demo/roles.json]: We have two demo users that gets created: end user and system user IIRC. Maybe add them?

**Comment by cobusc** [comment]
:+1:

File: authentication_service/demo/roles.json (hunk 1 above)
Comment (AltusBarry) [reply authentication_service/demo/roles.json]: Will add them locally, didn't want to introduce extra logic to add them to something we only added for the demo.

**Pull request closed without merging**